README.md

```diff
@@ -29,10 +42,10 @@ On Windows, you can use directory junctions instead of symlinks (symlinks requir
 ```powershell
 # Using PowerShell
 PS> git clone https://github.com/sublimehq/Packages.git
-PS> cmd /c mklink /J "$env:APPDATA/Sublime Text 3/Packages/Python" (convert-path ./Packages/Python)
+PS> cmd /c mklink /J "$env:APPDATA/Sublime Text/Packages/Python" (convert-path ./Packages/Python)
 ```
 
-Alternatively, download the portable version, and clone this repository directly as a subdirectory of the *Data* folder.
+Alternatively, download the portable version, and clone this repository directly as a subdirectory of the *`Data`* folder.
```

Comment:
If we tell them to use *Preferences > Browse Packages…* then we can remove this paragraph since it will work for portable also.